bukkit/src/main/java/org/popcraft/bolt/matcher/block/MultipleFacingMatcher.java

```diff
@@ -1,23 +1,35 @@
 package org.popcraft.bolt.matcher.block;
 
+import com.google.common.collect.Sets;
 import org.bukkit.Material;
 import org.bukkit.block.Block;
 import org.bukkit.block.BlockFace;
 import org.bukkit.block.data.MultipleFacing;
 import org.bukkit.entity.EntityType;
 import org.popcraft.bolt.matcher.Match;
+import org.popcraft.bolt.util.EnumUtil;
 
 import java.util.EnumSet;
 import java.util.HashSet;
 import java.util.Set;
 
-public class SkulkVeinMatcher implements BlockMatcher {
+public class MultipleFacingMatcher implements BlockMatcher {
     private static final EnumSet<BlockFace> CARTESIAN_FACES = EnumSet.of(BlockFace.NORTH, BlockFace.SOUTH, BlockFace.EAST, BlockFace.WEST, BlockFace.UP, BlockFace.DOWN);
+    // Future: make immutable (Set.of)
+    private static final Set<Material> MULTIPLE_FACING = Sets.newHashSet(Material.SCULK_VEIN, Material.GLOW_LICHEN);
```

Comment:
We can use the same trick here to initialize a mutable set
```suggestion
    private static final Set<Material> MULTIPLE_FACING = new HashSet<>(Set.of(Material.SCULK_VEIN, Material.GLOW_LICHEN));
```